internal/middleware/logger.go

```diff
@@ -21,13 +21,15 @@ func LoggerMiddleware(rootLogger *zerolog.Logger) func(next http.Handler) http.H
 			bytesIn, _ := strconv.Atoi(r.Header.Get("Content-Length"))
 			traceId := ctxval.TraceId(r.Context())
 			edgeId := ctxval.EdgeRequestId(r.Context())
+			corrId := ctxval.CorrelationId(r.Context())
 			lctx := rootLogger.With().
 				Timestamp().
 				Str("trace_id", traceId).
 				Str("remote_ip", r.RemoteAddr).
 				Str("url", r.URL.Path).
 				Str("method", r.Method).
-				Int("bytes_in", bytesIn)
+				Int("bytes_in", bytesIn).
+				Str("corr_id", corrId)
```

Comment:
If you choose correlation id as the term, then I would suggest to use the full field name just to be clear:

```suggestion
				Str("correlation_id", corrId)
```